<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CableDialog</class>
 <widget class="QDialog" name="CableDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kabel anlegen oder ändern</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="2" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>(typisch kleiner 1)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Bezugsquelle
(Link)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bemerkungen:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Restfehler</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditDealer"/>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelQuality">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string> +</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Hersteller</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QComboBox" name="comboBoxImpedance"/>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>* sqrt(f)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="labelK1">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="QTextEdit" name="textEditRemarks"/>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="labelK3">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="labelK2">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QComboBox" name="comboBoxDiameter"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>D(f):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditManufacturer"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Impedanz</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>*f +</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="calcButton">
     <property name="text">
      <string>Berechnen</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Stützstellen:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Durchmesser</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Bezeichnung</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditDataSheet"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Datenblatt
Dateiname oder Link</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CableDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CableDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
